src/app/map/[id]/components/Map.tsx

```diff
@@ -578,6 +576,7 @@ export default function Map({
                 const area = turf.area(feature);
                 const roundedArea = Math.round(area * 100) / 100;
                 insertTurf({
+                  id: crypto.randomUUID(),
```

Comment:
note: this should be using `import v4 as uuidv4 from "uuid"` and then `id: uuidv4()`